docs/fastdds/api_reference/spelling_wordlist.txt

```diff
@@ -94,6 +95,8 @@ mutexes
 myFilterFactory
 nackResponseDelay
 nackSupressionDuration
+netmask
+Netmask
```

Comment:
I think that if you include only the one with Capital letter, it allows both
Can you try locally removing `netmask`?

Reply:
I tried locally and it failed, so I guess that we should keep it